episodes/formative-assessment.md

```diff
@@ -73,231 +74,249 @@ Any instructional tool that generates feedback and is used in a formative way to
+spend the remainder reviewing and providing feedback on one another's assessments.
 
-Review the MCQ designed by your partner. When providing feedback, try to answer the following questions:
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::: callout
 
-- Is the question clear and easy to understand? Could the wording be improved in some way?
-- Are the incorrect answers to the MCQ plausible distractors?
-- Do the incorrect answers provide diagnostic power, to help an Instructor identify the misconception the learner has?
-- Are there any incorrect answers missing i.e. are there other misconceptions that could be detected with this MCQ?
+## Assessing Knowledge Before a Workshop
 
-Share your feedback in the collaborative notes document.
+The same approach to designing exercises within a lesson can also be used
+to create a short "re-assessment" questionnaire for potential learners
+to complete when they register for a workshop teaching the lesson
+(or for self-evaluation before following the lesson on their own).
+You can use [the list of prerequisite knowledge that you defined earlier](audience.md#defining-prerequisite-knowledge)
+to help with this.
 
+If you collect the results of this questionnaire,
+use it to follow up with people who have registered for the workshop
+but do not fit the intended target audience,
+to manage their expectations about how useful the workshop will be for them.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+You should aim to create all your assessments before you have written the explanatory content of your lesson (recall Nicholl's [backward design](https://en.wikipedia.org/wiki/Backward_design)). These assessments will guide your lesson design process by knowing exactly which knowledge you'd expect from your learners at any point in the lesson.
 
 
 :::::::::::::::::::::::::::::::::::::::: keypoints
 
-- The goal of lesson development is to ensure that the **attained curriculum** matches the **intended curriculum** as closely as possible.
 - Assessments are a way to determine whether the objectives you defined for the lesson have been reached.
+- Exercises help learners commit what they've learned to long-term memory.
+- Some types of exercises are better for particular audiences and to address certain objectives.
 - **Formative assessment** happens *during teaching* and provides feedback both to an instructor and a learner - about progress and whether learning of new concepts occurred but also about any misunderstandings and misconceptions which can hinder further learning.
-- It is important to detect misconceptions as early as possible and formative assessments (such as multiple choice questions) can help us with this.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-
-[^1]: See chapter 1, _How Does Students' Prior Knowledge Affect Their Learning?_,
-      of [Ambrose et al. 2010](learners/reference.md#litref).
```

Comment:
Think these lines still need to be included as it is referenced on lines 221 and 232